src/assets/styles/global.scss

```
@@ -110,6 +175,46 @@ a:hover {
     right: -30px;
 }
 
+/* Медиазапрос для мобильных устройств (ширина экрана до 767px) */
+@media (max-width: 767px) {
+    .prompt {
+        width: 90%;
+        /*  ширина рамки на мобильных устройствах */
+        margin: 20px auto;
+        /* Центр по горизонтали и  отступы */
+        padding: 10px;
+        /*  внутренние отступы */
+        border-radius: 20px;
+        /*  радиус скругления углов */
```

```suggestion
        /*  радиус скругления углов */
        display: flex;
        flex-direction: column;
```
Добавила флекс, чтобы лучше расположить нижние правые звёздочки через `align-self` (см ниже)